sender/sender.cpp

```diff
@@ -0,0 +1,23 @@
+
+#include <iostream>
+#include "../sensor/temperature.h"
+#include "../sensor/state_of_charge.h"
+#include <vector>
+#include "sender.h"
+
+using namespace std;
+
+bool sendSensorDataToConsole(){
+  bool isDataSendSuccessfully = false;
+  vector<int> temperatureValues = generateTemperatureValues(TEMPERATURE_MAX_VALUE ,TEMPERATURE_MIN_VALUE, READINGS_IN_STREAM);
+  vector<int> socValues = generateSocValues(SOC_MAX_VALUE ,SOC_MIN_VALUE, READINGS_IN_STREAM);
+
+  if((temperatureValues.size() == READINGS_IN_STREAM) && (socValues.size() == READINGS_IN_STREAM)) {
```

Comment:
Need to include check for the READINGS_IN_STREAM is greater than 0.

Reply:
READINGS_IN_STREAM is a preprocessor; the function will print the data only if the list size == 50